packages/backend/src/remote/activitypub/renderer/note.ts

```diff
@@ -111,6 +111,16 @@ export default async function renderNote(note: Note, dive = true, isTalk = false
 		...apemojis,
 	];
 
+	if (quote) {
+		tag.push({
+			type: 'Link',
+			rel: 'https://misskey-hub.net/ns#_misskey_quote',
```

Comment:
Seems to be a dead/invalid link? 
![image](https://user-images.githubusercontent.com/44733677/175430712-427bcf16-9800-458e-bf2e-9b5b0bb16484.png)

Reply:
no problem

Reply:
We are using URIs to make the namespace unique. So there is no problem if you can't access it.

Reply:
For completeness: This is as specified for ActivityPub vocabulary:
> ### `rel`
> A link relation associated with a [Link](https://www.w3.org/TR/activitystreams-vocabulary/#dfn-link). The value MUST conform to both the [[HTML5](https://www.w3.org/TR/html5/)] and [[RFC5988](https://datatracker.ietf.org/doc/html/rfc5988#section-4)] "link relation" definitions.
-- https://www.w3.org/TR/activitystreams-vocabulary/#dfn-rel

The "link relation" definition from RFC 5988 allows two relation types, relevant is this one:

> ### 4.2. Extension Relation Types
> Applications [...] can use an extension relation type, which is a URI [...]. Although the URI can point to a resource that contains a definition of the semantics of the relation type, clients SHOULD NOT automatically access that resource [...].
-- https://datatracker.ietf.org/doc/html/rfc5988#section-4.2